app/1.0/docs/devguide/data-system.md

```diff
@@ -526,7 +526,7 @@ A change notification event is fired when one of the following
 
 *   An array mutation.
 
-The event type follows a naming convention of <code><var>property</var>-changed</code>, where
+Event types follow a naming convention of <code><var>property</var>-changed</code>, where
```

Comment:
So this is an interesting one. The JavaScript `Event` object has a [type property](https://developer.mozilla.org/en-US/docs/Web/API/Event/type), which defines the well... type of event that occurred. "Event types" suggests that we're talking about all events, everywhere, but in this case we're talking Polymer's change notification events, and "The" refers back to "A change notification event" in the previous sentence.

This could be expanded to something like: "The event's type property indicates which property changed: it follows the naming convention of blahditty blah."